src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -673,16 +673,16 @@ void func_8087C9F8(EnHorse* this) {
 void func_8087CA04(EnHorse* this, GlobalContext* globalCtx) {
 }
 
-#ifdef NON_MATCHING
 void EnHorse_Init(Actor* thisx, GlobalContext* globalCtx2) {
     GlobalContext* globalCtx = globalCtx2;
     EnHorse* this = THIS;
+    Skin* skin = &this->skin;
 
     Actor_ProcessInitChain(&this->actor, sInitChain);
     EnHorse_ClearDustFlags(&this->dustFlags);
     D_801BDAA4 = 0;
     Skin_Setup(&this->skin);
-    this->riderPos = this->actor.world.pos;
+    this->riderPos = thisx->world.pos;
```

Comment:
`thisx` can be used all over this function, it's just this particular line that needs the `thisx` (`world.pos` specifically often needs `thisx` to lower/change the compiler managed stack). What's the protocol again when this is needed? Should I just use `thisx` on all instance of `this->actor`? Or just all instances of `world.pos`?

Reply:
Generally we do the same as OoT: if thisx is required in one place, and the function still matches when you replace it everywhere, that's preferred. Otherwise it's nice if there's some consistency, but if there can't be, never mind.